apps/bmd/src/pages/PollWorkerScreen.test.tsx

```diff
@@ -30,10 +29,59 @@ it('renders PollWorkerScreen', async () => {
       printer={fakePrinter()}
       tally={getZeroTally(election)}
       togglePollsOpen={jest.fn()}
+      enableLiveMode={jest.fn()}
     />
   )
 
-  advanceTimers()
-
   getByText('Polls are currently open.')
 })
+
+test('enabling Live Election Mode', async () => {
+  const election = electionSampleWithSeal as Election
+  const enableLiveMode = jest.fn()
+  const { getByText } = render(
+    <PollWorkerScreen
+      appPrecinctId={defaultPrecinctId}
+      ballotsPrintedCount={0}
+      election={election}
+      isPollsOpen
+      isLiveMode={false}
+      machineConfig={fakeMachineConfig({ appMode: VxMarkOnly })}
+      printer={fakePrinter()}
+      tally={getZeroTally(election)}
+      togglePollsOpen={jest.fn()}
+      enableLiveMode={enableLiveMode}
+    />
+  )
+
+  fireEvent.click(getByText('Switch to Live Election Mode'))
+  fireEvent.click(getByText('Cancel'))
+  expect(enableLiveMode).not.toHaveBeenCalled()
+
+  fireEvent.click(getByText('Switch to Live Election Mode'))
+  fireEvent.click(getByText('Yes'))
+  expect(enableLiveMode).toHaveBeenCalled()
+})
+
+test('cannot enable testing mode from live mode', async () => {
+  const election = electionSampleWithSeal as Election
+  const enableLiveMode = jest.fn()
+  const { getByText } = render(
+    <PollWorkerScreen
+      appPrecinctId={defaultPrecinctId}
+      ballotsPrintedCount={0}
+      election={election}
+      isPollsOpen
+      isLiveMode
+      machineConfig={fakeMachineConfig({ appMode: VxMarkOnly })}
+      printer={fakePrinter()}
+      tally={getZeroTally(election)}
+      togglePollsOpen={jest.fn()}
+      enableLiveMode={enableLiveMode}
+    />
+  )
+
+  expect(
+    (getByText('Switch to Testing Mode') as HTMLButtonElement).disabled
+  ).toBe(true)
```

Comment:
Should this option even be shown on the screen? (not a blocker for PR merge)

Reply:
My thinking was that it might be confusing to see the section, click the button, confirm the modal, then have the button completely disappear. But if the section and the warning are still there maybe it's okay? cc @benadida

Reply:
That makes sense, but then we're showing a UI which will never be enabled.

I suggest updating the modal copy to be clear that this is a one-way change (not a toggle).

After reading this text the user will be prepared to no longer see the previous button once they have confirmed the modal.

Reply:
What about this?

![127 0 0 1_3000_(Surface Go) (1)](https://user-images.githubusercontent.com/1938/98309078-6d98c180-1f7e-11eb-830d-440c2a7fc21a.png)

Reply:
This is great. Very clear.